### DISCUSSION TITLE
Education as a possible solution (with shortcomings)

### DISCUSSION THREAD

**Discussion by rh5140** [albertkun/21S-ASIAAM-191A#61] [Past Thinking Caps]
I like the idea of increasing awareness with more classes taught earlier. My high school actually used to require that we take a geography class to fulfill a graduation requirement. It would definitely be difficult to make the type of sweeping change in curriculum necessary to require that students learn critical cartography, but I know recently there was a successful movement pushing to require ethnic studies in California high schools, and there are people pushing for diversifying the books taught at my former school district, so it’s possible! I think it might also be a double-edged sword to require a class on mapping because while there would be good teachers, there will also be teachers who perpetuate harmful beliefs. For example, my high school geography course just reinforced the view that maps are, as Google’s English dictionary defines, “a diagrammatic representation of an area of land or sea showing physical features, cities, roads, etc.,” and I remember seeing a Tweet about how an ethnic studies course reinforced racial stereotypes. If a critical cartography course is taught through a colonialist lens, for example, it might do more harm than good by giving students the impression that they have learned enough to not marginalize people when there would still be serious gaps in their knowledge.

**Comment by ghost** [comment]
I think your take on this is very interesting and touches on a lot of thought-provoking points! There are drawbacks and shortcomings with most things but I know from personal experience that learning about mapping and the distribution of communities at an "earlier" age definitely helps to open eyes. I was able to take a human geography course my freshman year of high school and the teacher did a good job of covering the good and the bad going on in the world. Although it was not a cartography class maps were always at the center of lessons because of how efficiently (although not always correctly!) they can be used to display data, especially in geography. I'd say that class was pretty influential considering I am now a Geography major and GIS minor, but I can also think of so many things I wish would have been part of my mandated education, like ethnic studies. I recognize that my experience is not everyone else's which is why I think it is important to consider points like you brought up, concerning the possibility of the perpetuation of harmful narratives and misconceptions.
